numpy/linalg/_linalg.py

```diff
@@ -1741,6 +1741,40 @@ def svd(a, full_matrices=True, compute_uv=True, hermitian=False):
         return s
 
 
+def _svdvals_dispatcher(x):
+    return (x,)
+
+
+@array_function_dispatch(_svdvals_dispatcher)
+def svdvals(x, /):
+    """
+    Returns the singular values of a matrix (or a stack of matrices) ``x``.
+    When x is a stack of matrices, the function will compute the singular
+    values for each matrix in the stack.
+
+    This function is Array API compatible.
```

Comment:
Can you add a sentence explaining that this is the same thing as `linalg.svd` with `compute_uv=False`?

Reply:
Sure - I added a one-sentence explanation.